runtime/peaq-dev/src/lib.rs

```diff
@@ -149,10 +151,11 @@ pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 
 /// Maximum number of blocks simultaneously accepted by the Runtime, not yet included into the
 /// relay chain.
-pub const UNINCLUDED_SEGMENT_CAPACITY: u32 = 3;
+pub const UNINCLUDED_SEGMENT_CAPACITY: u32 = 2 * MAX_BLOCK_PROCESSING_VELOCITY + 1;
 /// How many parachain blocks are processed by the relay chain per parent. Limits the number of
 /// blocks authored per slot.
-pub const BLOCK_PROCESSING_VELOCITY: u32 = 1;
+/// 12 to achieve 500ms blocktime
+pub const MAX_BLOCK_PROCESSING_VELOCITY: u32 = 12;
```

Comment:
Why is different between here and peaq runtime?

Reply:
Let us have a call for that. In my opinion, we should make the major configs the same between peaq-dev and peaq to ensure the codeflow level is almost the same, but not only on the behavior level